command/CommandController.js

```diff
@@ -162,6 +171,18 @@ router.get('/search', VerifyToken, async function(req, res) {
   }
 });
 
+function logMetric(cmd, userid, token) {
```

Comment:
Should we send user-agent header from the initial request? This way we can track if call comes from Alexa (AlexaMediaPlayer/2.0.200535.0 (Linux;Android 5.1.1) ExoPlayerLib/1.5.9), iOS or other devices